### ISSUE TITLE
Action button out of the box support for vue material design icons

### ISSUE THREAD

**Issue by PVince81** [nextcloud/nextcloud-vue#1613]
As I understood, we want to move to material design icons as per https://github.com/nextcloud/nextcloud-vue/issues/1051

Currently when using icons from the vue material design icons library, the style doesn't fit:

This is an action button inside a menu (on my branch https://github.com/nextcloud/spreed/pull/4569):

```
				<ActionButton
					v-shortkey="['h']"
					:close-after-click="true"
					@shortkey="toggleHandRaised"
					@click="toggleHandRaised">
					<Hand
						slot="icon"
						class="action-button__icon"
						:size="24"
						fill-color="#000000"
						decorative
						title="" />
					{{ raiseHandButtonLabel }}
				</ActionButton>
```
where Hand comes from `import Hand from 'vue-material-design-icons/Hand'`

With the above, the menu looks like this:
<img width="118" alt="image" src="https://user-images.githubusercontent.com/277525/100468450-bdc5f800-30d4-11eb-8400-b725c3830bbf.png">

it is not centered, probably because of background-size not being usable with that svg element.

I tried bumping its size to 44px to match the other icon sizes but then the hand becomes huge:
<img width="118" alt="image" src="https://user-images.githubusercontent.com/277525/100468529-dfbf7a80-30d4-11eb-913c-87bff9b95e49.png">

In any case, as a consumer I shouldn't have to tinker with the styles.
Ideally we could define some styles that are based on the generated "material-design-icon" SVG.

**Comment by PVince81** [comment]
it appears that setting "vertical-align: middle" on the svg centers it correctly.

so we should add a rule for it.

**Comment by PVince81** [comment]
I guess even better would be to have a property `:material-design-icon` to provide a name and the required extra attributes are all taken care of.

**Comment by PVince81** [comment]
PR here for a quick fix: https://github.com/nextcloud/nextcloud-vue/pull/1614

we need it because the ActionButton consumer cannot override the styles with `::v-deep` since the popover itself might be in the document body

**Comment by juliushaertl** [comment]
> I guess even better would be to have a property :material-design-icon to provide a name and the required extra attributes are all taken care of.

I talked about this with @ma12-co in the past and I cannot see a way we can make this happen without bundling all material design icons in the library. What we would need is a way to let the library keep those as as es6 imports so that they would be bundled by the app in the end somehow, but I could figure out any way to do so. 

Though I'd say that is something for a separate issue if we really want to tackle it at some point.

**Comment by juliushaertl** [comment]
Fixed by #1614
